lib/grpc-api-assistant/grpc_steps.rb

```diff
@@ -6,9 +6,9 @@
   @stored_known_values[name] = Helpers.get_env(name) || name + "_" + SecureRandom.uuid
 end
 
-Given /^the value '(.+)' is saved in a key called '(.+)'$/ do |value, name|
+Given /^the value '(.*)' is saved in a key called '(.+)'$/ do |value, name|
   if value.downcase == "time.now"
-    value = Time.now.strftime("%s")
+    value = Time.now.utc.iso8601
```

Comment:
`Time.now.strftime("%s")` is not properly formatted for gRPC requests and will return an error. `Time.now.utc.iso8601` will allow the string to be properly formatted.

Reply:
this seems reasonable.  Since we're changing this line of code, I think we should also change the format to match the new `RANDOM()` operator.  I'm not too worried about breaking changes, but we could do it in a non-breaking way.

Reply:
I'm not really sure if I understood exctly what you meant here but I tried to take my interpretation of this? Let me know if this isn't what you meant by avoiding non-breaking changes.